tests/RuleFilterParser.Tests/LinqTests.cs

```diff
@@ -183,6 +227,32 @@ select new
         //Assert.Equal(1, persons.Count);
         Assert.Equal(5, persons.Count);
     }
+    
+    [Fact(Skip = "Does not work, see https://github.com/bcc-code/rule-filter-parser-dotnet/issues/26")]
```

Comment:
Isn't the query below in the format that _should_ work (ref #26)?
Perhaps queries in the non-supported format should result in an exception where the message also indicates the solution?

Reply:
Sorry, I used the wrong ticket: https://github.com/bcc-code/rule-filter-parser-dotnet/issues/29#issuecomment-1582278210 is correct.

> The current synchronous implementation I made executes the `Select` operation through `Enumerable.Select(..)` and does not perform the select transformation itself. This works fine for synchronous calls, but since C# by default does not support async. operations for Linq, this gets more overcomplicated.
> 
> ..

Reply:
I implemented now a client-side async. implementation of `Select`. The unit test works now.